examples/chat/chat_server/lib/src/generated/channel.dart

```diff
@@ -8,6 +8,8 @@
 // ignore_for_file: type_literal_in_constant_pattern
 // ignore_for_file: use_super_parameters
 
+// ignore_for_file: invalid_use_of_visible_for_testing_member
```

Comment:
Question: Are we sure we want to nuke this for the whole file? Would it be helpful for us to note if we add dependencies toward testing members in the generated code?

Reply:
I was mainly thinking we are using file-level scopes for all other linting rules. The same reasoning could apply to all of those: maybe we actually did a mistake somewhere. But in the end I chose this for consistency reasons.